virttest/qemu_vm.py

```diff
@@ -903,20 +903,29 @@ def add_memorys(devices, params):
             mem_params = params.object_params("mem")
             mem_params.setdefault("automem", "no")
             automem = mem_params["automem"] == "yes"
+
+            normalize_data_size = utils_misc.normalize_data_size
+            mem_size_m = "%sM" % mem_params["mem"]
+            mem_size_m = float(normalize_data_size(mem_size_m))
+            if mem_params.get("guest_max_mem"):
```

Comment:
Shall we use another name to express that it is a restriction set by user? How about `guest_mem_limit` or `vm_mem_limit`?